### DISCUSSION TITLE
what tool is used in readme second screenshot

### DISCUSSION THREAD

**Discussion by ivanbalashov239** [hyprwm/Hyprland#3915] [Q&A]
![image](https://github.com/hyprwm/Hyprland/assets/5340859/0989da3f-4db6-4c9a-808a-f40aed510e9a)

what is this?

**Comment by vaxerski** [comment]
eww (ags better tho)

**Comment by ErrorNoInternet** [comment]
[eww](https://github.com/elkowar/eww), rice is from https://github.com/end-4/dots-hyprland/tree/hybrid

**Comment by tarunjain1st** [comment]
how to add image to readme using img path
